addons/zylann.hterrain/tools/importer/importer_dialog.gd

```diff
@@ -40,6 +43,12 @@ func _ready():
 			"enum_items": ["Little Endian", "Big Endian"],
 			"enabled": false
 		},
+		"bit_depth": {
+			"type": TYPE_INT,
+			"usage": "number",
+			"number_items": [[RAW_16_BIT, "16-bit"], [RAW_24_BIT, "24-bit"]],
```

Comment:
Not sure about adding this fancy feature to the inspector script just for this, makes more things to review... it would have worked just as fine with a Godot-style enum with index values. Besides, this is an enum, not a "number". I would have rather extended the enum usage rather than creating a brand new thing.

Reply:
OK, I'll revise on weekend

Reply:
> I would have rather extended the enum usage 

Please check newest version and tell me if this is what you meant with extending enum usage